bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/transform/UpConverter.scala

```diff
@@ -142,6 +142,21 @@ case class LastStepConverter(semester: Semester) extends SemesterConverter {
  * This converter will upgrade to 2018B.
  */
 case object SemesterConverter2018ATo2018B extends SemesterConverter {
+  // REL-3363: No GMOS-N + Altair offered for 2018B. Remove Altair from component.
+  // The name of the instrument is auto-generated from its properties, so we don't worry about the <name>...</name> tag.
```

Comment:
Is that so? In some cases we do transform the name, are you sure it isn't necessary in this case?

Reply:
In the cases I tested, it wasn't necessary, but I may have missed a case.

Are there any instances in where we will upconvert just the XML and not actually create the blueprints from it?

Reply:
I'm going to go ahead and add the name change as well, but unfortunately, it's going to be a bit of a hassle, I think.

Reply:
Okay, so not nearly as bad as expected.